script/Config/App.lua

```diff
@@ -1,7 +1,11 @@
 Config.app = 'LTheoryRedux'
 
 Config.gameTitle   = "Limit Theory Redux"
-Config.gameVersion = "v0.007"
+Config.gameVersion = "v0.008"
```

Comment:
I don't think we should be updating this for every PR. Just leave it as it was, then at some point we will write some code to update this automatically

Reply:
I'd like to leave a placeholder for the actual game, but I can turn this into a generic reference we'll deal with later -- thanks for the suggestion.

Reply:
Yup, my main point is that if we have more than one person working on game code, people are going to conflict on updating the game version. Best to just leave it at v0.008 for a while and instead rely on the build date as the version (which will be in the title bar of the game window).